ejerciciosProg16-5/suma2Array.ts

```diff
@@ -1,8 +1,8 @@
 let botonaso = document.getElementById("comenzar");
 
 function verificacionNumero(numLocal: number): boolean {
-  let datoValido: boolean = isNaN(numLocal);
-  if (numLocal < 1 || datoValido === true) {
+  let datoValido: boolean;
+  if (numLocal < 1) {

```

Comment:
`let datoValido: boolean = isNaN(numLocal);`
Ahi estamos diciendo que el dato ES VALIDO si es un Not a Number... cosa que no es correcto.

Simplemente podemos validar que el numero NO sea menor que 1.